metrics/metricstest/metrics_mock.go

```diff
@@ -0,0 +1,118 @@
+package metricstest
+
+import (
+	"github.com/prebid/prebid-cache/config"
+	"github.com/prebid/prebid-cache/metrics"
+	"time"
+)
+
+// Define Mock metrics
```

Comment:
Please remove comment. It's associated with the MockHistograms var.

Reply:
done